Lib/test/test_mmap.py

```diff
@@ -1165,6 +1165,35 @@ def test_flush_parameters(self):
             m.flush(PAGESIZE)
             m.flush(PAGESIZE, PAGESIZE)
 
+    @unittest.skipUnless(hasattr(mmap, 'MAP_ANONYMOUS'), 'requires MAP_ANONYMOUS')
+    def test_set_name(self):
+        # Test setting name on anonymous mmap
+        m = mmap.mmap(-1, PAGESIZE)
+        self.addCleanup(m.close)
+        result = m.set_name('test_mapping')
+        self.assertIsNone(result)
```

Comment:
Would it be possible to add a functional test? Read /proc/pid/maps and check if it contains the string.

Reply:
For the Linux version, you can use `@requires_linux_version(x, y, z)` decorator. What do you mean by configuration? The feature can be disabled when Linux is configured?

Reply:
As I said before this feature is not enabled for Fedora by default even the kernel version is satisfied

Reply:
See: https://bugzilla.redhat.com/show_bug.cgi?id=2302746

Reply:
> Other thing: Do you know how to inject -X dev while running unittest through CI explicitly?

Yes, run tests in a subprocess.